00_find_bad_subs.py

```diff
@@ -0,0 +1,70 @@
+"""finds bad subjects and saves them as json.
```

Comment:
see my comment below: I think saving as TSV makes more sense

Also: I usually formulate the top level (module) docstring as a full sentence in imperative mode as well, for style